homeassistant/components/apsystemsapi_local/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""The config_flow for APsystems local API integration."""
+
+from aiohttp import client_exceptions
+from APsystemsEZ1 import APsystemsEZ1M
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_IP_ADDRESS, CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, LOGGER, UPDATE_INTERVAL
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_IP_ADDRESS): str,
+        vol.Required(CONF_NAME): str,
```

Comment:
Are you not able to get the name from the device? How is it shown in their app ? If the app has a static name, we should adapt that too.

Reply:
The name would be the serial ID, but that's it. Should it still be used?

Reply:
`CONF_NAME` is set in the config flow by the user. If we have a serial ID, it is great to include for the unique ID of device + config entry.